packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -368,10 +370,14 @@ exports.appellantSubmissionInformation = async (req, res) => {
 		summaryListData.sections.push(sectionView);
 	}
 
+	const css = fs.readFileSync(path.resolve(__dirname, '../public/stylesheets/main.css'), 'utf8');
```

Comment:
Might be nicer to do https://expressjs.com/en/starter/static-files.html rather than dealing with fs yourself (this should shut codeql up)

Reply:
This particular file of css is being passed to njk as a variable, so not sure how to use express.static in this case - isn't it used at root with app.use?  Or can you use express.static in the controller?

Reply:
and linking to it doesn't work? Does it need to be inline?